addons/hr_work_entry/models/resource_calendar.py

```diff
@@ -6,10 +6,17 @@
 class ResourceCalendar(models.Model):
     _inherit = 'resource.calendar'
 
-    # Override the method to add 'attendance_ids.work_entry_type_id.is_leave' to the dependencies
-    @api.depends('attendance_ids.work_entry_type_id.is_leave')
+    # Override the method to add 'attendance_ids.work_entry_type_id.category' to
+    # the dependencies
+    @api.depends('attendance_ids.work_entry_type_id.category')
     def _compute_hours_per_week(self):
         super()._compute_hours_per_week()
 
     def _get_global_attendances(self):
-        return super()._get_global_attendances().filtered(lambda a: not a.work_entry_type_id.is_leave)
+        global_attendances = super()._get_global_attendances()
+
+        def filter_leave_attendances(attendance):
+            category = attendance.work_entry_type_id.category
+            return category in [False, 'working_time']
+
+        return global_attendances.filtered(filter_leave_attendances)
```

Comment:
```suggestion
        return global_attendances.filtered_domain([('work_entry_type_id.category', '!=', ['absence'])])
```
what do you think of this ? :)